views/lib/components/files-storage/files-card.hbs

```diff
@@ -11,7 +11,7 @@
 			tabindex="0"
 			aria-label="{{$t "files.label.clickToOpenFile" (dict "name" this.name)}}"
 			role="button">
-			<div class="card-title" data-testid="file-title-card">
+			<div class="card-title" data-testid="file-title-card-{{@index}}">
```

Comment:
I think the `@index` keyword is only available on `#each` iterations. And `#each` loop is not used this file at all.

Reply:
yes `#each` used in the parent component `views/lib/components/files-storage/files-grid.hbs`